service/statistics/Events.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
     /**
-     * FIXME: needs documentation.
+     * Notifies for audio level in rtp statistics by ssrc.
```

Comment:
Thank you for adding documentation!

When documenting events, I'd like to have documentation on the parameters carried by the event as well. Otherwise, one has to search through the source code itself in order to determine what arguments and in what order the listener function is to expect.